src/base/serialiser.h

```diff
@@ -72,7 +73,7 @@ template <typename... Contexts> class Serialisable
     static SerialisedValue fromVectorToTable(const std::vector<T> &vector, Lambda getName)
     {
         SerialisedValue group;
-        for (auto it = vector.rbegin(); it < vector.rend(); it++)
+        for (auto it = vector.begin(); it < vector.end(); it++)
```

Comment:
Minor thing, but is there a good reason for not using a range here?
```suggestion
        for (const auto &value : vector)
```

Reply:
Range is just fine.  This was just a legacy from when we went through the vector in reverse order.